CHANGELOG.md

```diff
@@ -1,27 +1,222 @@
+- Desktop version personal settings now support custom JVM configuration with memory usage control to within 1 GB.
+- Desktop version supports exporting data files larger than 4 GB.
+- Optimized the en-US language wording of the product.
+
+### Bug fixes
+
+Connection Session
+
+- Sessions failing to remove after expiration, causing potential leaks [#2125](https://github.com/oceanbase/odc/pull/2125)
+- References to expired connection sessions failed to be promptly cleared, leading to resource leaks, which may cause increased memory consumption [#1914](https://github.com/oceanbase/odc/pull/1914)
+- Under high-frequency usage scenarios, executing SQL or viewing table data may encounter issues where the interface becomes unresponsive [#1797](https://github.com/oceanbase/odc/pull/1797)
+- Intermittent 404 errors when attempting to open SQL console [#1809](https://github.com/oceanbase/odc/pull/1809)
+
+SQL Execution
+
+- Commit/rollback button state not in sync with actual transaction status in OceanBase 4.2 [#2097](https://github.com/oceanbase/odc/pull/2097)
+- SQL statements with single-line comments unable to execute [#2085](https://github.com/oceanbase/odc/pull/2085)
+- Incorrect offsets for last SQL command without a trailing delimiter [#1970](https://github.com/oceanbase/odc/pull/1970)
+- Incompatibility of anonymous block execution module with Oracle 11g [#1759](https://github.com/oceanbase/odc/pull/1759)
+- DBMS output failed to output spaces [#1051](https://github.com/oceanbase/odc/issues/1970)
+- `#` and `$` character disappeared after SQL window formatted the code [#1490](https://github.com/oceanbase/odc/issues/1490)
+- Auto-complete in SQL window not available for MySQL datasource [#1718](https://github.com/oceanbase/odc/issues/1718)
+
+Result-set
+
+- Degrading performance when modifying multiple rows concurrently in result-set [#2007](https://github.com/oceanbase/odc/pull/2007)
+- In OceanBase MySQL mode, the datetime data type loses precision when displayed [#1996](https://github.com/oceanbase/odc/pull/1996)
+- Switching between viewing BLOB field text and hexadecimal images may cause the UI to freeze [#300](https://github.com/oceanbase/odc/issues/300)
 
+Table Object
 
+- Disparity in column names order between Index and Constraint views [#1948](https://github.com/oceanbase/odc/pull/1948)
+- Inability to view table details for MySQL version 5.6 [#1635](https://github.com/oceanbase/odc/pull/1635)
+- Sofa ODP table detail view not accessible [#2043](https://github.com/oceanbase/odc/pull/2043)
+- Unable to change NOT NULL fields to NULL in table structure editor [#1441](https://github.com/oceanbase/odc/issues/1441)
+- When partitioned table has multiple maximum values, only one is displayed [#1501](https://github.com/oceanbase/odc/issues/1501)
+- Button to delete table primary key is greyed out and not clickable [#1874](https://github.com/oceanbase/odc/issues/1874)
```

Comment:
### Table Objects

- The column order in index and constraint views is inconsistent. [#1948](https://github.com/oceanbase/odc/pull/1948)
- Unable to view table details for MySQL v5.6. [#1635](https://github.com/oceanbase/odc/pull/1635)
- Unable to view table details for Sofa ODP. [#2043](https://github.com/oceanbase/odc/pull/2043)
- Cannot change NOT NULL fields to NULL in table schema editing. [#1441](https://github.com/oceanbase/odc/issues/1441)
- When a partitioned table has multiple maximum values, only one maximum value is displayed. [#1501](https://github.com/oceanbase/odc/issues/1501)
- The button to delete table primary keys is grayed out and cannot be clicked. [#1874](https://github.com/oceanbase/odc/issues/1874)